src/python/pants/core/subsystems/python_bootstrap.py

```diff
@@ -89,7 +89,9 @@ class PythonBootstrapSubsystem(Subsystem):
 
     class EnvironmentAware(Subsystem.EnvironmentAware):
         search_path = StrListOption(
-            default=["<PYENV>", "<PATH>"],
```

Comment:
this was gruesome to debug but this ends up broken if we refer to `AsdfPathString.STANDARD` directly. You end up with:
```
--python-bootstrap-search-path="[<binary-paths>, <binary-paths>, ...]"
PANTS_PYTHON_BOOTSTRAP_SEARCH_PATH
search_path
    default: [
        "<PYENV>",
        "<ASDF>",
        "<PATH>"
    ]
    current value: [
        "<PYENV>",
        "AsdfPathString.STANDARD",
        "<PATH>"
    ]
```
For these str-Enum compositions, the `str` returns the fully qualified name:
```
str(AsdfPathString.STANDARD) == 'AsdfPathString.STANDARD'
```
but the `__format__()` method used in f-strings returns the intended `'<ASDF>'`